kit/src/main/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMIntrinsic.java

```diff
@@ -57,7 +57,8 @@
     private Object sourceSceneGraphObject;
 
     
-    FXOMIntrinsic(FXOMDocument document, GlueElement glueElement, Object targetSceneGraphObject,  List<FXOMProperty> properties) {
+    FXOMIntrinsic(FXOMDocument document, GlueElement glueElement, Object targetSceneGraphObject,
```

Comment:
Okay, good to know, didn't update it because this change is formatting only (following Checkstyle advice). Good to know, I will consider this for future commits.